deploy.env

```diff
@@ -66,11 +75,12 @@ GOTRUE_ADMIN_PASSWORD=password
 # If you are using a different domain, you need to change the redirect_uri in the OAuth2 configuration
 # Make sure that this domain is accessible to the user
 # Make sure no endswith /
-API_EXTERNAL_URL=http://your-host
+# Replace with your host name instead of localhost
+API_EXTERNAL_URL=http://localhost:${GOTRUE_PORT}
 
 # In docker environment, `postgres` is the hostname of the postgres service
 # GoTrue connect to postgres using this url
-GOTRUE_DATABASE_URL=postgres://supabase_auth_admin:${SUPABASE_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
+GOTRUE_DATABASE_URL=postgres://supabase_auth_admin:${SUPABASE_PASSWORD}@${GOTRUE_DATABASE_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
```

Comment:
Currently there is no way to run gotrue and appflowy on separate database. Hence, there is no good reason to have two separate host configuration.

Reply:
Noted